script.js

```diff
@@ -0,0 +1,79 @@
+/* add todo*/
+const todoInput=document.getElementById("todo-input");
+const addTodoButton=document.getElementById("todo-button");
+const doingTodoList=document.getElementById("doing");
+const doneTodoList=document.getElementById("done");
+let todoArray=[];
+const doingCount=document.getElementById("doing-count");
+const doneCount=document.getElementById("done-count");
+
+/*create todo li*/
+function createTodoList(todoObj){
+    const todoListItem=document.createElement("li");
+    const deleteImg=document.createElement("img");
+    todoListItem.innerText="🐈 "+todoObj.text;
+    todoListItem.id=todoObj.id;
+    todoListItem.addEventListener("click", toggleTodo);
+    deleteImg.src="./img/bin.png";
+    deleteImg.addEventListener("click", deleteTodo);
+    todoListItem.appendChild(deleteImg);
+    todoObj.type? doingTodoList.appendChild(todoListItem):doneTodoList.appendChild(todoListItem);
```

Comment:
삼항연산자를 사용하신 점이 인상깊네요 ㅎㅎ

다만 제 개인적인 생각으로는 삼항연산자는 대입을 하는 데 있어서는 가독성을 높여줄 수 있는데, 
여기처럼 조건에 따른 분기 동작이라면 if 문이 오히려 가독성이 좋을 수 있을 것 같습니다!
```suggestion
const isCompleted = (todo) => todo.type;

if (isCompleted(todo)) {
  doingTodoList.appendChild(todoListItem);
} else {
  doneTodoList.appendChild(todoListItem);
}
```

또 `type`라는 속성명만 두기보다 할일이 완료되었는지를 확인하는 `isCompleted`와 같은 함수를 통해서 추상화해보는 것도 좋을 것 같습니다 ㅎㅎ

Reply:
오오 isCompleted라고 쓰는게 더 가독성이 좋을거 같네요ㅎㅎ 감사합니다!